README.md

```diff
@@ -542,9 +555,39 @@ A complete example can be found under "resources/test/test-config.json"
                 "key": "AccessKeyId",
                 "secret": "S3_STORAGE_SECRET_ACCESSKEYID"
             }
+        },{
+            "dataset": "s3-parquet-read",
+            "storageType": "S3",
+            "stripProps": true,
+            "parquet": {
+                "flushThreshold": 20,
+                "schema": "message test_schema { required int64 age; optional binary id (STRING);}"
+            },
+            "decode": {
+                "defaultNamespace": "_",
+                "namespaces": {
+                    "_": "http://data.io/foo/bar/"
+                },
+                "idProperty": "age",
+                "ignoreColumns": [
+                    "__index_level_0__"
+                ]
+            },
+            "props": {
+                "bucket": "s3-test-bucket",
+                "endpoint": "http://localhost:8888",
+                "region": "us-east-1",
+                "key": "AccessKeyId",
+                "secret": "S3_STORAGE_SECRET_ACCESSKEYID"
+            }
         }
     ]
 }
+```
 
+When reading parquet files it add a property called ```__index_leve_0__``` which is an AUTO_INCREMENT propery. There is added support for removing this column. There is also support for adding muiltiple fields in the ```ignoreColumns``` list if yod'd wish to filter out more fields.
```

Comment:
a few minor spelling mistakes; "adds a property", "index_level_0", "property", "you'd"